posthog/api/test/test_dashboard.py

```diff
@@ -1,20 +1,46 @@
 import json
 
 from dateutil import parser
+from django.db import connection
 from django.utils import timezone
 from django.utils.timezone import now
 from freezegun import freeze_time
 from rest_framework import status
 
 from posthog.models import Dashboard, Filter, Insight, Team, User
 from posthog.models.organization import Organization
-from posthog.test.base import APIBaseTest
+from posthog.test.base import APIBaseTest, QueryMatchingTest, snapshot_postgres_queries
 from posthog.utils import generate_cache_key
 
 
-class TestDashboard(APIBaseTest):
+class TestDashboard(APIBaseTest, QueryMatchingTest):
     CLASS_DATA_LEVEL_SETUP = False
 
+    @snapshot_postgres_queries
+    def test_retrieve_dashboard_list(self):
+        d1 = Dashboard.objects.create(team=self.team, name="a dashboard", created_by=self.user)
```

Comment:
non-blocking

Is there a REST API we can use to create the dashboards? This way to can test only at the REST API boundary level and not infer any internal details.